docs/submitting-applications.md

```diff
@@ -218,6 +218,115 @@ These commands can be used with `pyspark`, `spark-shell`, and `spark-submit` to
 For Python, the equivalent `--py-files` option can be used to distribute `.egg`, `.zip` and `.py` libraries
 to executors.
 
+# VirtualEnv for PySpark
```

Comment:
@zjffdu, mind if I ask to describe this is an experimental feature and it's very likely to be unstable and it's still evolving?